tests/test-recipes/metadata/build_number/run_test.bat

```diff
@@ -6,5 +6,5 @@ echo "%condalist%"
 if not "%condalist%"=="conda-build-test-build-number-1.0-1" exit 1
 cat "%PREFIX%\conda-meta\conda-build-test-build-number-1.0-1.json"
 if errorlevel 1 exit 1
-cat "%PREFIX%\conda-meta\conda-build-test-build-number-1.0-1.json" | grep '"build_number": 1'
+cat "%PREFIX%\conda-meta\conda-build-test-build-number-1.0-1.json" | FINDSTR /r /C:"\"build_number\": 1"
```

Comment:
Incidentally, this whole test could be written in Python. (e.g. https://github.com/conda/conda-build/blob/master/tests/test-recipes/metadata/extra_freeform_metadata/run_test.py)